public/app/features/plugins/admin/pages/PluginDetails.test.tsx

```diff
@@ -24,18 +25,28 @@ jest.mock('@grafana/runtime', () => {
   return mockedRuntime;
 });
 
-const renderPluginDetails = (pluginOverride: Partial<CatalogPlugin>): RenderResult => {
+const renderPluginDetails = (pluginOverride: Partial<CatalogPlugin>, pageId = PluginTabIds.OVERVIEW): RenderResult => {
   const plugin = getCatalogPluginMock(pluginOverride);
   const { id } = plugin;
-  const props = getRouteComponentProps({ match: { params: { pluginId: id }, isExact: true, url: '', path: '' } });
+  const props = getRouteComponentProps({
+    match: { params: { pluginId: id }, isExact: true, url: '', path: '' },
+    queryParams: { page: pageId },
+    location: {
+      hash: '',
+      pathname: `/plugins/${id}`,
+      search: `?page=${pageId}`,
+      state: undefined,
+    },
+  });
   const store = configureStore({
     plugins: getPluginsStateMock([plugin]),
   });
 
   return render(
     <Provider store={store}>
       <PluginDetailsPage {...props} />
-    </Provider>
+    </Provider>,
+    { wrapper: MemoryRouter }
```

Comment:
**Question:** is this only "syntactic sugar" over actually writing out `<MemoryRouter>...</MemoryRouter>`?

Reply:
Yep, happy to change it to `<MemoryRouter>...</MemoryRouter>` if you think it's easier to read?

Reply:
No mate, it's absolutely fine like this, was just curious :)